packages/relay/src/lib/hbarlimiter/index.ts

```diff
@@ -19,7 +19,10 @@
  */
 
 import { Logger } from 'pino';
-import { Counter, Gauge, Registry } from 'prom-client';
+import constants from '../constants';
+import { Hbar, HbarUnit } from '@hashgraph/sdk';
+import { predefined } from '../errors/JsonRpcError';
+import { Registry, Counter, Gauge } from 'prom-client';
 import { formatRequestIdMessage } from '../../formatters';
 
 export default class HbarLimit {
```

Comment:
This class is going to be deprecated and removed with the hbar limiter redesign initiative, this story is supposed to add preemptive limiting logic in the new `HbarLimitService`.

Reply:
This PR mainly focuses on improving current preemtive rate limit logic. This [ticket#2937](https://github.com/hashgraph/hedera-json-rpc-relay/issues/2937) will implement the logic in the new `HbarLimitService` class.

Reply:
Then remove it from hbar limiter redesign epic and remove the "[HBAR Rate Limit Redesign]" prefix in the name of the issue.

Reply:
Yep that's a good point! updated!